forms/Form.php

```diff
@@ -142,6 +142,11 @@ class Form extends RequestHandler {
 	protected $extraClasses = array();
 
 	/**
+	 * @var array $default_classes The default classes to apply to the Form
```

Comment:
Can you pop a `@config` notifier here if it's configurable?

We haven't, but probably should add this to the coding conventions.